package.json

```diff
@@ -49,6 +49,7 @@
     "yargs": "^8.0.1"
   },
   "devDependencies": {
+    "aurelia-webpack-plugin": "^2.0.0-rc.5",
```

Comment:
I see that's a RC version, is it stable enough (I don't know Aurelia at all)?

Reply:
IMO, yes. Aurelia has a solid track record of releasing RC's that rarely have many breaking changes. 2.0 of this plugin has not been released but 1.x has been abandoned. They tend to run RC's for a while to be certain that their true x.0 release is reliable after being battle tested a bit :)

Reply:
When 2.0 is released, I'll do my best to remember to come back and update this though :)